rskj-core/src/test/java/co/rsk/peg/performance/VoteFeePerKbChangeTest.java

```diff
@@ -72,7 +73,8 @@ void voteFeePerKbChange() throws VMException {
     void voteFeePerKbChange_unauthorized() throws VMException {
         BridgeStorageProviderInitializer storageInitializer = Helper.buildNoopInitializer();
 
-        Coin genesisFeePerKB = BridgeRegTestConstants.getInstance().getGenesisFeePerKb();
+        FeePerKbConstants feePerKbMainNetConstants= FeePerKbMainNetConstants.getInstance();
```

Comment:
i think i'd do it this way:

```
        BridgeConstants bridgeConstants = BridgeMainNetConstants.getInstance();
        FeePerKbConstants feePerKbMainNetConstants = bridgeConstants.getFeePerKbConstants();
```

because the idea is to access the fee per kb constants only through the bridge constants

Reply:
(and remember to leave spaces around `=`)